include/mxx/distribution.hpp

```diff
@@ -111,15 +111,18 @@ void stable_distribute(_InIterator begin, _InIterator end, _OutIterator out, con
         size_t prefix = mxx::exscan(local_size, comm);
```

Comment:
`prefix` may be the same as `total_size` if the processes with rank lower than the current process have all the input elements.